Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -118,53 +122,128 @@ def adjoint(self,x, out=None):
         raise NotImplementedError
     
     @staticmethod
-    def PowerMethod(operator, iterations, x_init=None):
-        '''Power method to calculate iteratively the Lipschitz constant
-        
-        :param operator: input operator
-        :type operator: :code:`LinearOperator`
-        :param iterations: number of iterations to run
-        :type iteration: int
-        :param x_init: starting point for the iteration in the operator domain
-        :returns: tuple with: L, list of L at each iteration, the data the iteration worked on.
-        '''
+    def PowerMethod(operator, max_iteration=10, initial=None, tolerance = 1e-5,  verbose=False,  **deprecated_args):

```

Comment:
`verbose` parameter makes me think I'll get some printed feedback, while it is a parameter that makes return more stuff than just the largest eigenvalue. As it wasn't part of previous signature, I suggest  to use a more appropriate name, for instance `return_all`